README.md

```diff
@@ -2,20 +2,175 @@
 
 > Test client for Xo-Server
 
-## Usage
+## Setup
+
+- Xo-server needs to be installed on `localhost:9000`.  
+Otherwise, you should modifiy the `xoServerUrl` in the `congig.toml`.
+
+- You should have a `admin@admin.net` account with password `admin`.
+
+- There should be a Xen Server available at `192.168.1.3` with the credentials `root` and password `qwerty`.
 
-xo-server needs to be installed on `localhost:9000`.
-You should also have a `admin@admin.net` account with password `admin`.
+## Adding a test
 
-There should be a Xen Server available at `192.168.1.3`with the credentials `root` and password `qwerty`.
+### Organization
 
-Warning ! This client will create and delete a lot of items (users, groups, servers ...) to test features
+Reminder: the xo commands are composed of a **XO object** and a *method* (**user**.*create*, **job**.*getAll*, **vm**.*delete*,...). 
```

Comment:
I thought it was better to specify it, not because of `xo-cli` methods but for the organization of the tests for the moment. 

But you're right. I'll remove it :+1: